ontology/cdno.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: ontology_detail
+id: cdno
+title: Crop Dietary Nutrition Ontology
+jobs:
+  - id: https://travis-ci.org/Southern-Cross-Plant-Science/cdno
+    type: travis-ci
+build:
+  checkout: git clone https://github.com/Southern-Cross-Plant-Science/cdno.git
+  system: git
+  path: "."
+contact:
+  email: l.andres.hernandez.18@student.scu.edu.au
+  label: Liliana Andres Hernandez
+  github: 
+description: CDNO provides structured terminologies to describe nutritional attributes of crop plants and their harvested materials that contribute to human diet. 
+domain: dietary nutritional components, nutritional component concentrations
```

Comment:
Please use `diet, metabolomics and nutrition` -> we try to converge all nutrition related ontologies there.

Reply:
Ok, thanks, I will change this.